CMakeLists.txt

```diff
@@ -67,13 +51,21 @@ if (BUILD_EXAMPLES)
 	add_subdirectory(examples)
 endif()
 
+# library interface
+file(GLOB_RECURSE headers_library "${TAPKEE_INCLUDE_DIR}/*.hpp")
+add_library(tapkee_library INTERFACE)
+target_include_directories(tapkee_library INTERFACE ${TAPKEE_INCLUDE_DIR} "${EIGEN3_INCLUDE_DIR}")
```

Comment:
Could you put `TAPKEE_INCLUDE_DIR` into quotes so it don't fail on directories with spaces.